.golangci.yaml

```diff
@@ -104,12 +104,9 @@ linters-settings:
           recommendations:
           - fmt
           - errors
-  depguard:
-    list-type: denylist
-    include-go-root: false
-    packages-with-error-message:
-      - k8s.io/utils/pointer: "Use github.com/samber/lo ToPtr instead"
-      - golang.org/x/net/context: "use regular context instead"
+      - k8s.io/utils/pointer:
+          recommendations:
+          - github.com/samber/lo
```

Comment:
What is the intention to remove linting on `golang.org/x/net/context`? This is not included in other linting rules.

Reply:
Re-added, got missed when copy-pasting between the N nearly-identical files :facepalm: